### ISSUE TITLE
Wrong way to get Model Class

### ISSUE THREAD

**Issue by webard** [mateusjunges/trackable-jobs-for-laravel#29]
In Trackable.php file, line 22 there is:
```php
$this->trackedJob = TrackedJob::create([
            'trackable_id' => $this->model->id ?? $this->model->uuid,
            'trackable_type' => get_class($this->model),
            'name' => class_basename(static::class),
        ]);
```

but should be:
```php
$this->trackedJob = TrackedJob::create([
            'trackable_id' => $this->model->id ?? $this->model->uuid,
            'trackable_type' => $this->model->getMorphClass(),
            'name' => class_basename(static::class),
        ]);
```

This way, package will be respect Relation::morphMap setting.

**Comment by mateusjunges** [comment]
Thank you @webard for reporting and sorry for the delay. I'll fix it tomorrow first thing in the morning.

**Comment by mateusjunges** [comment]
@webard fixed and released in v1.5.1. Again, sorry for my late response, I'm very busy the last few days
